### ISSUE TITLE
Full story pages: How to skip posts that are of a certain category?

### ISSUE THREAD

**Issue by mhulse** [jekyll/jekyll-help#31]
Hi,
### TL;DR:

On my full story pages (permalink pages), I'd like to setup the previous/next pagination so that it skips any posts that have a category of X.
### Long version:

Specifically, I have some posts that are categorized as "external", and when these posts are clicked, the end user is taken to that external link.

Using the previous/next post pagination, I'd like to ignore all external posts and just paginate between actual posts.

I build my Jekyll site locally, so plugins aren't an issue, but I was mostly curious if there's built-in way to "filter" the pagination object based on post frontmatter (like category)?

Does any of that make sense?

Again, this is the pagination for the post pages … The permalink pages. Not the index page pagination.

For now, I don't use the prev/next pagination on permalink pages because I couldn't figure out how to ignore my "external" posts.

Thanks!
